lib/Gemfile

```diff
@@ -42,7 +42,12 @@ gem "rufus-lru",            "~>1.0.3",        :require => false
 gem "uuidtools",            "~>2.1.3",        :require => false
 gem "sass",                 "3.1.20",         :require => false
 gem "trollop",              "~>2.0",          :require => false
-gem "psych",                "~>2.0.12"
+
+# JRuby has its own version of psych
+if RUBY_PLATFORM != 'java'
+  gem "psych",              "~>2.0.12"
+end
```

Comment:
@tenderlove @djberg96 I do see psych 2.0.14.pre1 (jruby) - have we been able to test this / move forward on this item?